### ISSUE TITLE
Failed to deploy to my own cloud

### ISSUE THREAD

**Issue by pythen** [openhab/openhab-google-assistant#223]
I cannot deploy the current revision to my own cloud instance.
I start the deployment with the command:

`gcloud beta functions deploy openhabGoogleAssistant --runtime nodejs10 --trigger-http`

and it’s ends with this error message:

```
Deploying function (may take a while - up to 2 minutes)...failed.
ERROR: (gcloud.beta.functions.deploy) OperationError: code=3, message=Function failed on loading user code. This is likely due to a bug in the user code.
```

I can find the following error message in the cloud log file:

```
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant Provided module can't be loaded.
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant Did you list all required modules in the package.json dependencies?
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant Detailed stack trace: Error: Cannot find module '../../package.json'
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Function.Module._load (internal/modules/cjs/loader.js:562:25)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Module.require (internal/modules/cjs/loader.js:692:17)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at require (internal/modules/cjs/helpers.js:25:18)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Object.<anonymous> (/workspace/devices/default.js:2:24)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Module._compile (internal/modules/cjs/loader.js:778:30)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Module.load (internal/modules/cjs/loader.js:653:32)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
2021-06-08 11:26:58.819 MESZopenhabGoogleAssistant at Function.Module._load (internal/modules/cjs/loader.js:585:3)
2021-06-08 11:26:58.821 MESZopenhabGoogleAssistant Could not load the function, shutting down.
```

Do you have any idea what this could be?

**Comment by michikrug** [comment]
Did you run a `npm i` in the `functions` folder?

**Comment by michikrug** [comment]
Just saw that this might indeed be a bug

**Comment by michikrug** [comment]
Should be fixed with 14e9f728991d9c1ff6a7db6bb3f6304f2cffc41f

**Comment by pythen** [comment]
Yes, it works now.

Thank you very much, Michael.
